### ISSUE TITLE
PREDEPLOYMENT_CALIB_DATE

### ISSUE THREAD

**Issue by BrianKingNOC** [euroargodev/sensor_metadata_json#14]
At the moment

PREDEPLOYMENT_CALIB_DATE

is not an Argo variable in the Argo manual

calibration date is often included in 
PREDEPLOYMENT_CALIB_COMMENT

but in a non-uniform way.

It would be much better to introduce 
PREDEPLOYMENT_CALIB_DATE

**Comment by SBS-EREHM** [comment]
Agreed.  I think this is "putting our foot down" on some absolutely mandatory PREDEPLOYMENT metadata that should be part of the Argo User's Guide.

**Comment by BrianKingNOC** [comment]
Closing this because a wider issue has been started "PREDEPLOYMENT_CALIB_DATE, SENSOR_FIRMWARE_VERSION and other names that aren't in the Argo users manual and perhaps should be"
